v2.2/cluster-setup-troubleshooting.md

```diff
@@ -6,202 +6,433 @@ toc: true
+
+**Solution:** To successfully join the node to the cluster, start the node again, but this time include the `--certs-dir` flag
+
+### Certification expiration
+
+If you’re running a secure cluster, be sure to monitor your certificate expiration. If one of the inter-node certificates expires, nodes will no longer be able to communicate which can look like a network partition.
+
+To check the certificate expiration date:
+
+1. [Access the Admin UI](admin-ui-access-and-navigate.html#access-the-admin-ui).
+2. Click the gear icon on the left-hand navigation bar to access the **Advanced Debugging** page.
+3. Scroll down to the **Even More Advanced Debugging** section. Click **All Nodes**. The **Node Diagnostics** page appears. Click the certificates for each node and check the expiration date for each certificate in the Valid Until field.
+
+### Client password not set
+
+While connecting to a secure cluster as a user, CockroachDB first checks if the client certificate exists in the `cert` directory. If the client certificate doesn’t exist, it prompts for a password. If password is not set and you press Enter, the connection attempt fails, and the following error is printed to stderr:
+
 ~~~
-node belongs to cluster {"cluster hash"} but is attempting to connect to a gossip network for cluster {"another cluster hash"}
+Error: pq: invalid password  
+Failed running "sql"
 ~~~
 
-**Solution**: Disassociate the node from the existing directory where you've stored CockroachDB data. For example, you can do either of the following:
+**Solution:** To successfully connect to the cluster, you must first either generate a client certificate or create a password for the user.
 
-- Choose a different directory to store the CockroachDB data:
+## Clock sync issues
 
-    {% include copy-clipboard.html %}
-    ~~~ shell
-    # Store this node's data in [new directory]
-    $ cockroach start --store=<new directory> --join=<cluster host> <other flags>
-    ~~~
+### Node clocks are not properly synchronized
 
-- Remove the existing directory and start a node joining the cluster again:
+See [What happens when node clocks are not properly synchronized](https://www.cockroachlabs.com/docs/dev/operational-faqs.html#what-happens-when-node-clocks-are-not-properly-synchronized)
 
-    {% include copy-clipboard.html %}
-    ~~~ shell
-    # Remove the directory
-    $ rm -r cockroach-data/
-    ~~~
+How can I tell how well node clocks are synchronized?
 
-    {% include copy-clipboard.html %}
-    ~~~ shell
-    # Start a node joining the cluster
-    $ cockroach start --join=<cluster host>:26257 <other flags>
-    ~~~
+See [How can I tell how well node clocks are synchronized](https://www.cockroachlabs.com/docs/dev/operational-faqs.html#how-can-i-tell-how-well-node-clocks-are-synchronized)
+
+## Capacity planning issues
+
+Possible issues:
```

Comment:
Rather than simply "possible issues," typically you expect some kind lead-in that helps the user orient themselves to the information you're trying to present.